.github/workflows/test.yml

```diff
@@ -51,3 +51,8 @@ jobs:
 
     - name: Install pandas-stubs and run tests on the installed stubs
       run: poetry run poe test_dist
+
+    - if: matrix.python-version == '3.10' && matrix.os == 'ubuntu-latest'
+      name: Run pytest (against pandas nightly)
+      continue-on-error: true
```

Comment:
this seems to hide errors but I don't think there is a warning/yellow option

Reply:
When we were doing testing in the microsoft stubs project, we wanted to ignore warnings from pyright, and Graham Wheeler (owner of the repo) modified some github action to consume the pyright results and ignore the warnings.

Maybe there is an equivalent with pytest???

Reply:
Warnings are easier to control (--disable-warnings and with filterwarnings). In our case, we want to see the warnings. I didn't find an option to always return 0.

Reply:
That's why he created the action to consume errors and warnings (it was from pyright), and then return 0 if there were only warnings and no errors.

Reply:
I think you need condition: succeededOrFailed() on later steps. This is how we do never fail prerelease on statsmodels.

Reply:
We only have it on one step after pytest so might not be a good idea here.

Reply:
You can also have || true after the pytest command to make a never fail.

Reply:
> You can also have || true after the pytest command to make a never fail.

That would be the least invasive option, but at least the following doesn't work:

`run: poetry run poe pytest --nightly ; exit 1 || true`

Reply:
Since `pytest` is being run from a python script, couldn't you get or parse the results from running pytest within python, and then determine the exit status that gets returned?

Reply:
I just did something like that :)